app/src/interfaces/files/files.vue

```diff
@@ -211,7 +215,7 @@ const fields = computed(() =>
 const page = ref(1);
 
 const query = computed<RelationQueryMultiple>(() => ({
-	fields: fields.value,
+	fields: [...fields.value, 'directus_files_id.filename_download'],
```

Comment:
`directus_files_id` can't be hardcoded, as the name of the field used in the relationship isn't guaranteed. Instead, we should update the `computed` `fields` above to include the correct field name from the `relationInfo` ref 👍🏻

Reply:
I hope I had understood you correctly, this would be my adjustment:

```
const fields = computed(() =>
	adjustFieldsForDisplays(
		[...getFieldsFromTemplate(templateWithDefaults.value), `${relationInfo.value?.relation.field}.filename_download`],
		relationInfo.value?.junctionCollection.collection ?? ''
	)
);
```

if that is correct, i will commit the change.

Reply:
LGTM!